### ISSUE TITLE
Connection Error

### ISSUE THREAD

**Issue by jojofofo36** [XenorPLxx/playnite-library-psn#7]
I cannot connect to my account, i tried without 2FA security but same thing happen.
<img width="356" alt="psn" src="https://user-images.githubusercontent.com/114771853/197689926-ce2ca140-276a-4022-aa57-3ad8528ef9fc.png">

**Comment by jojofofo36** [comment]
didnt see the other post

**Comment by leandromacedo** [comment]
I'm still having this problem. Could you kindly share what was your solution? I tried returning using the left arrow but it didn't work.

**Comment by jojofofo36** [comment]
> I'm still having this problem. Could you kindly share what was your solution? I tried returning using the left arrow but it didn't work.

https://github.com/XenorPLxx/playnite-library-psn/issues/5#issuecomment-1288236965 

i just did this and it worked for me

**Comment by risbac** [comment]
Hi ! Tried this workaround with the arrow, but it's not working here. Still the same error :-( Tried to delete the token file, to clear the cache, restart, but still the same.

**Comment by MostDefiantly** [comment]
I am also running into the same, and apparently this has been an issue off and on for GOG and other plugin/browser extensions that hit the Sony site for various tasks.

Here's a link on what appears to be the problem: https://github.com/FriendsOfGalaxy/galaxy-integration-psn/issues/7#issuecomment-644759063

And a workaround from the GOG PSN plugin, which I can't find a facsimile in the Token.JSON this plugin uses: https://github.com/FriendsOfGalaxy/galaxy-integration-psn/issues/7#issuecomment-645286024

TL;DR from the various threads I have been reading about the GOG plugin:

"They say it's due to the embedded browser in Galaxy being old, which leads to a "CORS error".

The reason it works sometimes is because if you luck out and get a direct connection to a PSN server, PSN's servers don't care about this issue and will let your request through.

If your request goes to Akamai's load balanacer before PSN, Akamai's filters will throw the CORS error and reject your request."
